cfme/infrastructure/pxe.py

```diff
@@ -530,25 +509,75 @@ def update(self, updates, cancel=False):
             msg = 'Customization Template "{}" was saved'.format(name)
         main_view.flash.assert_success_message(msg)
 
-    def delete(self, cancel=True):
+
+@attr.s
+class CustomizationTemplateCollection(BaseCollection):
+    """Collection class for CustomizationTemplate"""
+
+    ENTITY = CustomizationTemplate
+
+    def create(self, name, description, image_type, script_type, script_data, cancel=False):
+        """
+        Creates a Customization Template object
+
+        Args:
+            cancel (boolean): Whether to cancel out of the creation.  The cancel is done
+                after all the information present in the CT has been filled in the UI.
+            name: Name of CT
+            description:description: The description field of CT.
+            image_type: Image type of the CT.
+            script_data: Contains the script data.
+            script_type: It specifies the script_type of the script.
+        """
+
+        if image_type is None:
+            image_type = '<Choose>'
+
+        elif script_type is None:
+            script_type = '<Choose>'
```

Comment:
**REQUIRED**
Not needed. Widgetastic automatically skips fields with `None` value.